examples/scaffolding/contrib/DeepResearch/run_deep_research.py

```
@@ -0,0 +1,67 @@
+    parser.add_argument("--model", type=str, default="gpt-oss-20b")
+    return parser.parse_args()
+
+
+async def main():
+    args = parse_arguments()
+    client = AsyncOpenAI(api_key=args.openai_api_key, base_url=args.base_url)
+
+    generation_worker = OpenaiWorker(client, args.model)
+    generation_worker.register_task_handler(ChatTask, chat_handler)
+
+    mcp_worker = await MCPWorker.init_with_urls([f"http://0.0.0.0:8082/sse"])
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)